internal/engine/buildcontrol/build_control_test.go

```diff
@@ -29,6 +30,37 @@ func TestNextTargetToBuildDoesntReturnCurrentlyBuildingTarget(t *testing.T) {
 	f.assertNoTargetNextToBuild()
 }
 
+func TestPodsDisableLocalScheduling(t *testing.T) {
```

Comment:
This test title is v confusing to me b/c I'm not used to thinking of k8s manifests as "pods". Maybe something like: `TestCurrentlyBuildingK8sResourceDisablesLocalScheduling` and consider renaming the k8s manifests

Reply:
done